src/components/screens/AddAssetScreen/AddAssetBottomSheetContent.tsx

```diff
@@ -1,125 +1,276 @@
+            publicAddress={account?.publicKey ?? ""}
+            hasBorder={false}
+            hasBackground={false}
+          />
+          <Text md primary>
+            {account?.accountName}
+          </Text>
+        </View>
+      ),
+      titleColor: themeColors.text.secondary,
+    },
+  ];
+
+  if (!isMalicious && !isSuspicious) {
+    listItems.push({
+      icon: <Icon.Globe01 size={16} color={themeColors.foreground.primary} />,
+      title: t("network"),
+      trailingContent: (
+        <Text md primary>
+          {network === NETWORKS.PUBLIC ? t("mainnet") : t("testnet")}
+        </Text>
+      ),
+      titleColor: themeColors.text.secondary,
+    });
+  }
+
+  if (isMalicious || isSuspicious) {
+    listItems.push({
+      icon: <Icon.Circle size={16} color={themeColors.foreground.primary} />,
+      title: t("addAssetScreen.assetAddress"),
+      trailingContent: (
+        <View className="flex-row items-center gap-2">
+          <TouchableOpacity
+            onPress={() => {
+              copyToClipboard(asset.issuer);
+            }}
+            className="p-1"
+          >
+            <Icon.Copy01 size={16} color={themeColors.foreground.primary} />
+          </TouchableOpacity>
+          <TouchableOpacity
+            onPress={() => {
+              copyToClipboard(asset.issuer);
+            }}
+          >
+            <Text md primary>
+              {getDisplayText()}
+            </Text>
+          </TouchableOpacity>
+        </View>
+      ),
+      titleColor: themeColors.text.secondary,
+    });
+  }
+
   return (
-    <View className="flex-1 justify-center items-center mt-2">
-      <AssetIcon
-        token={{
-          type: asset.assetType as AssetTypeWithCustomToken,
-          code: asset.assetCode,
-          issuer: {
-            key: asset.issuer,
-          },
-        }}
-      />
+    <View className="flex-1 justify-center items-center mt-8">
+      <View>
+        <AssetIcon
+          token={{
+            type: asset.assetType as AssetTypeWithCustomToken,
+            code: asset.assetCode,
+            issuer: {
+              key: asset.issuer,
+            },
+          }}
+        />
+        {isMalicious && (
+          <View className="absolute -bottom-1 -right-1 rounded-full p-1 z-10 bg-red-3">
+            <Icon.AlertCircle size={12} color={themeColors.status.error} />
+          </View>
+        )}
+      </View>
+
       <View className="mt-4" />
-      <Text lg primary medium>
+      <Text lg primary>
         {asset.assetCode}
       </Text>
+
       <View className="mt-1" />
       {asset.domain && (
         <Text sm secondary>
           {asset.domain}
         </Text>
       )}
-      <View className="mt-2" />
+
+      <View className="mt-4" />
       <Badge
         variant="secondary"
         size="md"
-        icon={<Icon.ShieldPlus size={16} />}
+        icon={
+          !isMalicious && !isSuspicious ? (
+            <Icon.PlusCircle size={14} />
```

Comment:
I think we should always use the PlusCircle here, Figma is probably incorrect

Reply:
https://www.figma.com/design/KwkHXQxbNmDllwermJtnRu?node-id=5576-68703#1370550809